ms/applink.c

```diff
@@ -132,6 +132,15 @@ OPENSSL_Applink(void)
     return OPENSSL_ApplinkTable;
 }
 
+extern __declspec(dllexport)
+void 
+# if defined(__BORLANDC__)
+__stdcall
+# else
+__cdecl
+# endif
+OPENSSL_SetApplink(void** (*custom)());
```

Comment:
> What that break borland-c builds?

For reference, we no longer support Borland C builds in sense that you can't compile OpenSSL with Borland C. Reference to `__BORLANDC__` in ms/applink.c is about using .dlls compiled with MSVC with *application* compiled with Borland C.

> Would it be appropriate to just remove the BORLANDC test?

But why is it even dllexport? It's the one application would call, not provide. But let's put aside this for now...